_data/simple-icons.json

```diff
@@ -17728,7 +17728,7 @@
 		"guidelines": "https://www.wappalyzer.com/logos"
 	},
 	{
-		"title": "Warner Bros.",
+		"title": "Warner Bros",
 		"slug": "warnerbros",
```

Comment:
Nope. If the icon data object has a `slug` property, the `slug` is used instead.